include/EUTelGeometryTelescopeGeoDescription.tcc

```diff
@@ -0,0 +1,50 @@
+#ifndef EUTELGEOMETRYTELECOPEGEODESCRIPTION_TCC  
+#define EUTELGEOMETRYTELECOPEGEODESCRIPTION_TCC
+
+/** Determine id of the sensor in which point is locate
+ *  * 
+ *  * @param globalPos 3D point in global reference frame
+ *  * @return sensorID or -999 if the point in outside of sensor volume
+ *  */
+//MUST OUTPUT -999 TO SIGNIFY THAT NO SENSOR HAS BEEN FOUND. SINCE USED IN PATTERN RECOGNITION THIS WAY.
+template<class number>
+int EUTelGeometryTelescopeGeoDescription::getSensorID( number globalPos[] ) const {
```

Comment:
A template was neater I thought. If we overload then we will have to write the same function again right?